### PR TITLE
Kush-dev-challenge-01

### PR DESCRIPTION
**Pull request by bhavygarg**

### CODE REVIEW

File: docs/dev-profile/kush-dev-profile.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      <i>
+        <a href="https://github.com/bhavygarg" target="_blank">
+          Bhavy Garg GIT <br
+        /></a>
+      </i>
+    </div>
+    <div class="name">
+      <img
+        class="avatar"
+        src="https://media-exp1.licdn.com/dms/image/C4E03AQHZWxE-vzBkPg/profile-displayphoto-shrink_800_800/0/1606599299980?e=1617840000&v=beta&t=nFMBkFvRTphgpq7LZWzgnb7_MkY78KYPpAPGWVK6_FI"
```
Comment (barrycumbie) [inline docs/dev-profile/kush-dev-profile.html:66]: remember to use a base64 converter to embed the picture & not a link to an online resource.

File: docs/dev-profile/kush-dev-profile.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    </div>
+
+    <div class="gitmeme">
+      <h4>Meme:</h4>
+      <img
+        class="memePic"
+        src="http://2.bp.blogspot.com/-jxG7CDnHIog/UphAJGfPHuI/AAAAAAAAAIA/G_qEX9FEFeY/s1600/computer+science+major.jpg"
+        alt="gitMene"
+      />
+    </div>
```
Comment (barrycumbie) [inline docs/dev-profile/kush-dev-profile.html:89]: where's the Nu checker link?

**Comment by barrycumbie** [comment]
@bhavygarg 👏🏼👏🏼👏🏼👏🏼

Very close. Remember to include in the comments the link to your active test site & the Nu checker. See #20 as an example. 

A couple of comments on the code: add Base64 encoding & link to Nu

File: docs/dev-profile/kush-dev-profile.html (hunk 1 above)
Comment (bhavygarg) [reply docs/dev-profile/kush-dev-profile.html]: I have sent you new pull request with base 64 added to the file.

File: docs/dev-profile/kush-dev-profile.html (hunk 2 above)
Comment (bhavygarg) [reply docs/dev-profile/kush-dev-profile.html]: I will update Nu checker link as I will need the url from your Github repo.

**Comment by bhavygarg** [comment]
Active Link to site https://bhavygarg.github.io/dev-challenge-1/


> @bhavygarg 👏🏼👏🏼👏🏼👏🏼
> 
> Very close. Remember to include in the comments the link to your active test site & the Nu checker. See #20 as an example.
> 
> A couple of comments on the code: add Base64 encoding & link to Nu

**Comment by barrycumbie** [comment]
Looks good @bhavygarg 

just need the Nu link to the ```active test site``` and a Nu link on the page itself!

**Comment by barrycumbie** [comment]
didn't have to put the active test site & Nu to that on the actual page...but no big deal. It'll fly.

**Comment by bhavygarg** [comment]
Hello Dr. Cumbie. This is my nu validator link:
https://validator.w3.org/nu/?showsource=yes&doc=https%3A%2F%2Fbhavygarg.github.io%2Fdev-challenge-1%2F

**Comment by barrycumbie** [comment]
This is your active test site: 

https://bhavygarg.github.io/dev-challenge-01/dev-profile/kush-dev-profile.html

I'm not sure what this is:https://validator.w3.org/nu/?showsource=yes&doc=https%3A%2F%2Fbhavygarg.github.io%2Fdev-challenge-1%2F

Or this: 

https://bhavygarg.github.io/dev-challenge-1/

**Comment by barrycumbie** [comment]
What I mean is, 

you are submitting one file **FILE A** but then submitting a link to another file **FILE B**. I need you to get precise in your communication...send me the links for the file I am to merge into our product.

You are mixing up your REPOS: ```dev-challenge-1``` vs. ```dev-challenge-01```. mix-ups like this just slow everything down trying to figure it out. Syntax errors in code are understandable but mixing up folder & file names is careless.

**Comment by bhavygarg** [comment]
> What I mean is,
> 
> you are submitting one file **FILE A** but then submitting a link to another file **FILE B**. I need you to get precise in your communication...send me the links for the file I am to merge into our product.
> 
> You are mixing up your REPOS: `dev-challenge-1` vs. `dev-challenge-01`. mix-ups like this just slow everything down trying to figure it out. Syntax errors in code are understandable but mixing up folder & file names is careless.

Hey @barrycumbie Sorry about the confusion. I will just delete that repo `dev-challenge-1` for any  confusion. 

Here is the nu validator link:
https://validator.w3.org/nu/?showsource=yes&doc=https%3A%2F%2Fbhavygarg.github.io%2Fdev-challenge-01%2Fdev-profile%2Fkush-dev-profile.html

**Comment by barrycumbie** [comment]
Now we are moving along!

of course, still never game me the link directly to your active test site: https://bhavygarg.github.io/dev-challenge-01/dev-profile/kush-dev-profile.html 😠 making me have to copy/paste from the Nu. Oh well, I'll get over it. 

BUT... 

I am noticing on your ```.html``` page that there is not a Nu link there anymore. I thought you had these before and now they are gone. You need a link on this page:

https://bhavygarg.github.io/dev-challenge-01/dev-profile/kush-dev-profile.html

that I can click and check the validity
```
<a href=
"SOME URL HERE THAT AUTO CHECKS THIS PAGE:
   https://  [but remember, nor your username: bhavygarg 
  but will get changed to MY user name]
.github.io/dev-challenge-01/dev-profile/kush-dev-profile.html" 
target="_blank">check this page's validity with Nu!</a> 

```

**Comment by bhavygarg** [comment]
> Now we are moving along!
> 
> of course, still never game me the link directly to your active test site: https://bhavygarg.github.io/dev-challenge-01/dev-profile/kush-dev-profile.html 😠 making me have to copy/paste from the Nu. Oh well, I'll get over it.
> 
> BUT...
> 
> I am noticing on your `.html` page that there is not a Nu link there anymore. I thought you had these before and now they are gone. You need a link on this page:
> 
> https://bhavygarg.github.io/dev-challenge-01/dev-profile/kush-dev-profile.html
> 
> that I can click and check the validity
> 
> ```
> <a href=
> "SOME URL HERE THAT AUTO CHECKS THIS PAGE:
>    https://  [but remember, nor your username: bhavygarg 
>   but will get changed to MY user name]
> .github.io/dev-challenge-01/dev-profile/kush-dev-profile.html" 
> target="_blank">check this page's validity with Nu!</a> 
> ```

@barrycumbie 
This is my active site with nu validator link: https://bhavygarg.github.io/dev-challenge-01/dev-profile/kush-dev-profile.html

Sir, my file is still not available in your repo. so, for nu link validator, I've provided link from dev-challenge-01 fork repo.

**Comment by barrycumbie** [comment]
Fix line 123 when you submit dev-02

```diff
-<a href="https://validator.w3.org/nu/?showsource=yes&doc=https%3A%2F%2Fbhavygarg.github.io%2Fdev-challenge-01%2Fdev-profile%2Fkush-dev-profile.html" target="blank">Nu Link Validator to site.</a>
+<a href="https://validator.w3.org/nu/?showsource=yes&doc=https%3A%2F%2Fbarrycumbie.github.io%2Fdev-challenge-01%2Fdev-profile%2Fkush-dev-profile.html" target="blank">Nu Link Validator to site.</a>
```

**Comment by bhavygarg** [comment]
Hey @barrycumbie 
I updated my new validator link in my active site. Also, I added my dev profile link to index.html. 
Should I send you new pull request because I am already in collaborator. Please help in this issue.

This is my new active site: https://barrycumbie.github.io/dev-challenge-01/dev-profile/kush-dev-profile.html

**Pull request merged**